lib/utils/sdp-utils.js

```diff
@@ -12,6 +12,30 @@ const mergeSdpMedia = (sdp1, sdp2) => {
   return sdpTransform.write(parsedSdp1);
 };
 
+const isOpusFirst = (sdp) => {
+  const parsedSdp = sdpTransform.parse(sdp);
+  const idx = parsedSdp.media[0].rtp.findIndex((entry) => entry.codec === 'opus');
+  return idx === 0;
+};
+
+const opusFirst = (sdp) => {
```

Comment:
please rename this function to 'makeOpusFirst'

Reply:
fixed